[2024-01-16T16:29:11.966+0200] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_sql_pipeline.insertion_1 manual__2024-01-16T14:29:06.705753+00:00 [queued]>
[2024-01-16T16:29:11.971+0200] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_sql_pipeline.insertion_1 manual__2024-01-16T14:29:06.705753+00:00 [queued]>
[2024-01-16T16:29:11.971+0200] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-16T16:29:11.990+0200] {taskinstance.py:2192} INFO - Executing <Task(SqliteOperator): insertion_1> on 2024-01-16 14:29:06.705753+00:00
[2024-01-16T16:29:11.992+0200] {standard_task_runner.py:60} INFO - Started process 49739 to run task
[2024-01-16T16:29:11.995+0200] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'executing_sql_pipeline', 'insertion_1', 'manual__2024-01-16T14:29:06.705753+00:00', '--job-id', '129', '--raw', '--subdir', 'DAGS_FOLDER/sqlite_pipeline.py', '--cfg-path', '/tmp/tmpodklhyzk']
[2024-01-16T16:29:11.996+0200] {standard_task_runner.py:88} INFO - Job 129: Subtask insertion_1
[2024-01-16T16:29:12.036+0200] {task_command.py:423} INFO - Running <TaskInstance: executing_sql_pipeline.insertion_1 manual__2024-01-16T14:29:06.705753+00:00 [running]> on host DESKTOP-9NHC01K.
[2024-01-16T16:29:12.104+0200] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='executing_sql_pipeline' AIRFLOW_CTX_TASK_ID='insertion_1' AIRFLOW_CTX_EXECUTION_DATE='2024-01-16T14:29:06.705753+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-16T14:29:06.705753+00:00'
[2024-01-16T16:29:12.105+0200] {sql.py:276} INFO - Executing: 
            INSERT INTO users (name, age, is_active) VALUES 
                ('Julie', 30, false),
                ('Peter', 55, true),
                ('Emily', 37, false),
                ('Katrina', 54, false),
                ('Joseph', 27, true);
        
[2024-01-16T16:29:12.109+0200] {base.py:83} INFO - Using connection ID 'my_sqlite_cnn' for task execution.
[2024-01-16T16:29:12.124+0200] {base.py:83} INFO - Using connection ID 'my_sqlite_cnn' for task execution.
[2024-01-16T16:29:12.125+0200] {sql.py:450} INFO - Running statement: 
            INSERT INTO users (name, age, is_active) VALUES 
                ('Julie', 30, false),
                ('Peter', 55, true),
                ('Emily', 37, false),
                ('Katrina', 54, false),
                ('Joseph', 27, true);
        , parameters: None
[2024-01-16T16:29:12.125+0200] {sql.py:459} INFO - Rows affected: 5
[2024-01-16T16:29:12.138+0200] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=executing_sql_pipeline, task_id=insertion_1, execution_date=20240116T142906, start_date=20240116T142911, end_date=20240116T142912
[2024-01-16T16:29:12.168+0200] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-16T16:29:12.202+0200] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
